tests/unit/utils/Utils.h

```diff
@@ -55,4 +58,44 @@ SGStringList<T> generateRandomStringData(
 	return strings;
 }
 
+struct NeuralLayerTestUtil
+{
+	template <typename T>
+	static SGMatrix<T> create_rand_sgmat(
```

Comment:
I think this is not a function we should have in neuralnet unittests.
Rather, this should be somewhere where other random number generators sit, like `Random.h` or something more appropriate. Random normal matrices are used in a lot of places in shogun